quiche/include/quiche.h

```diff
@@ -520,6 +621,12 @@ typedef struct {
 
     // DATAGRAM frame extension parameter, if any.
     ssize_t peer_max_datagram_frame_size;
+
+    // The stats of the connection's paths.
+    quiche_path_stats paths[8];
```

Comment:
where does `8` come from?

Reply:
It is actually a magic number to ensure the structure can hold up to 8 path's statistics. I could instead put a simple pointer along with a length, but it is unclear to me whether the caller should allocate the required space or the `ffi` should instead do this.